tests/integration/jcloud/test_basic_app.py

```diff
@@ -38,11 +38,11 @@ async def _test_ws_route(app_id):
         await websocket.send(json.dumps({"interval": 1}))
 
         received_messages = []
-        for _ in range(5):
+        for _ in range(6):
             message = await websocket.recv()
             received_messages.append(message)
 
-        assert received_messages == ["0", "1", "2", "3", "4"]
+        assert received_messages[1:] == ["0", "1", "2", "3", "4"]
```

Comment:
Is this because the first message is the connection id?

Reply:
yea